lld/test/ELF/arm-thumb-jump8-11.s

```diff
@@ -0,0 +1,32 @@
+# REQUIRES: arm
```

Comment:
We have arm-thumb-narrow-branch-check.s . 

Another candidate name is arm-thumb-narrow-branch.s